### PR TITLE
Learning rate switching & fix indent

### PR DESCRIPTION
**Pull request by hdeezy**
Make learning rates switch from training textual inversion to unet/text encoder after unfreeze_lora_step. I think this is how it was explained in the paper linked(?)

Either way, it might be useful to add another parameter to activate unet/text encoder training at a certain step instead of at unfreeze_lora_step. This would let the user have more control.

Also fix indenting to make save_steps and logging work properly.

### CODE REVIEW

**Comment by hdeezy** [comment]
Maybe changing/lowering the textual inversion learning rate after unfreeze_lora_step is another option worth exploring.

**Comment by cloneofsimo** [comment]
thanks!
